SpatialGDK/Source/SpatialGDK/Public/Utils/RPCContainer.h

```diff
@@ -102,6 +102,7 @@ class SPATIALGDK_API FRPCContainer
 	void ProcessOrQueueRPC(const FUnrealObjectRef& InTargetObjectRef, ERPCType InType, SpatialGDK::RPCPayload&& InPayload);
 	void ProcessRPCs();
 	void DropForEntity(const Worker_EntityId& EntityId);
+	int32 GetNumQueuedRPCs() const { return QueuedRPCs.Num(); }
```

Comment:
Where is this used?

Reply:
No longer used. Was used by the receiver to inform the SpatialSpanIdStore to drop a certain number of SpanIds.